lib/main-window.js

```diff
@@ -2,12 +2,18 @@ const electron = require('electron')
 const app = electron.app
 const BrowserWindow = electron.BrowserWindow
 const path = require('path')
+const Config = require('electron-config')
+const config = new Config()
 
 var showMenu = process.platform !== 'win32'
+let windowSize = { width: 1080, height: 720 }
 
-var mainWindow = new BrowserWindow({
-  width: 1080,
-  height: 720,
+if(config.get('windowsize') === undefined) config.set('windowsize', windowSize)
```

Comment:
Please add space after `if` statement. We compliant [JavaScript Standard Style](https://github.com/feross/standard)!